staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -822,7 +832,8 @@ func UpdateResource(r rest.Updater, scope RequestScope, typer runtime.ObjectType
 		if wasCreated {
 			status = http.StatusCreated
 		}
-		responsewriters.WriteObject(status, scope.Kind.GroupVersion(), scope.Serializer, result, w, req.Request)
+
+		transformResponseObject(ctx, scope, req, res, status, result)
```

Comment:
This is actually a `writeObject` call which may optionally transform?  Seems misnamed  and perhaps in the wrong spot.  Why not place in the `responsewriters` package?